### PR TITLE
Update onnx batch norm parser  

### PR DESCRIPTION
**Pull request by anwu1219**
Generalize batch norm to high dimension inputs.

### CODE REVIEW

File: maraboupy/MarabouNetworkONNX.py [hunk 1]
```diff
@@ -443,22 +444,30 @@ def batchNorm(self, node):
...
+
+        numChannels = len(scales)
 
         # Get variables
-        inputVars = self.varMap[inputName].reshape(-1)
-        outputVars = self.makeNewVariables(nodeName).reshape(-1)
-        assert len(inputVars) == len(outputVars)
+        inputVars = self.varMap[inputName].reshape(numChannels, -1)
+        outputVars = self.makeNewVariables(nodeName).reshape(numChannels, -1)
+        assert(inputVars,shape == outputVars.shape)
```
Comment (tagomaru) [inline maraboupy/MarabouNetworkONNX.py]: Probably, inputVars.shape?

File: maraboupy/MarabouNetworkONNX.py [hunk 2]
```diff
@@ -443,22 +444,30 @@ def batchNorm(self, node):
...
+        for i in range(numChannels):
+            for j in range(numInputs):
+                # Add equation
+                # To know this computation,
+                # refer to https://github.com/onnx/onnx/blob/master/docs/Operators.md#batchnormalization.
+                e = MarabouUtils.Equation()
+                e.addAddend(-1, outputVars[i][j])
+                e.addAddend(1 / np.sqrt(input_variances[i] + epsilon) * scales[i], inputVars[i][j])
+                e.setScalar(input_means[i] / np.sqrt(input_variances[i] + epsilon) * scales[i] - biases[i])
+                self.addEquation(e)
```
Comment (tagomaru) [inline maraboupy/MarabouNetworkONNX.py:470]: It looks like you are trying to expand it to multi channels rather than multi dimensions.

Comment (tagomaru) [reply maraboupy/MarabouNetworkONNX.py:470]: Sorry, never mind this.
Input is flattened by `self.varMap[inputName].reshape(numChannels, -1)`.

**Review by tagomaru — CHANGES_REQUESTED** [review]

File: maraboupy/MarabouNetworkONNX.py (hunk 1 above)
Comment (anwu1219) [reply maraboupy/MarabouNetworkONNX.py]: Yep, fixed..

**Review by tagomaru — APPROVED** [review]

**Pull request merged**